services/src/handlers/plots.rs

```diff
@@ -0,0 +1,225 @@
+use serde::{Deserialize, Serialize};
+use snafu::ResultExt;
+use uuid::Uuid;
+use warp::http::Response;
+use warp::Filter;
+
+use geoengine_datatypes::primitives::{BoundingBox2D, SpatialResolution, TimeInterval};
+use geoengine_operators::engine::{QueryRectangle, TypedPlotQueryProcessor};
+
+use crate::contexts::Context;
+use crate::error;
+use crate::handlers::authenticate;
+use crate::ogc::util::{parse_bbox, parse_time};
+use crate::users::session::Session;
+use crate::util::parsing::parse_spatial_resolution;
+use crate::workflows::registry::WorkflowRegistry;
+use crate::workflows::workflow::WorkflowId;
+
+#[derive(Debug, Clone, PartialEq, Deserialize, Serialize)]
+pub(crate) struct GetPlot {
```

Comment:
can't you just use `QueryRectangle`?

Reply:
I don't know since I am deserializing from URL parameters. I guess a custom deserializer for the string value in one field for a whole qrect is quite complicated and doesn't make the API better. If we would deserialize from JSON, I would agree.